examples/warntoggle/mqttwarn/customfunctions.py

```diff
@@ -0,0 +1,36 @@
+import time
+import copy
+import json
+
+def togglestate(topic, payload, section, srv): 
+    filename = "warntoggle.json"
+    default_topicblock = False
+
+    try:
+        with open(filename) as infile:    
+            toggles = json.load(infile)
+            infile.close()
+
+        if topic in toggles: 
+            # file found, topic found
+            topicblock = toggles[topic]
+	    srv.logging.debug('togglestate() was called from the ' + section + ' section and found ' + topic + ' in ' + filename)
```

Comment:
I think there's an indentation problem here. Is that a tab?

Reply:
Yep. Corrected.